### PR TITLE
logs should all be lower case when possible

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Standardize logs to be lower cased 
Bonus: Fixed a func comment to start with the func name 

Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: pkg/nfd-worker/nfd-worker.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ import (
 	"sigs.k8s.io/node-feature-discovery/source/local"
 	"sigs.k8s.io/node-feature-discovery/source/memory"
 	"sigs.k8s.io/node-feature-discovery/source/network"
-	"sigs.k8s.io/node-feature-discovery/source/panic_fake"
+	panicfake "sigs.k8s.io/node-feature-discovery/source/panic_fake"
```
Comment (marquiz) [inline pkg/nfd-worker/nfd-worker.go:51]: nit: this change is unrelated and unnecessary

**Review by marquiz — COMMENTED** [review]
One nit, again referring to [K8s commit message guidelines](https://www.kubernetes.dev/docs/guide/pull-requests/#commit-message-guidelines) and *[the first word in the commit message subject should be capitalized unless it starts with a lowercase symbol or other identifier](https://www.kubernetes.dev/docs/guide/pull-requests/#the-first-word-in-the-commit-message-subject-should-be-capitalized-unless-it-starts-with-a-lowercase-symbol-or-other-identifier)*, specifically.

You add a context, for example (you can come up with something better):
```
logging: start messages with lower case
```

File: pkg/nfd-worker/nfd-worker.go (hunk 1 above)
Comment (ArangoGutierrez) [reply pkg/nfd-worker/nfd-worker.go:51]: this was my IDE to blame, sorry will fix

Comment (ArangoGutierrez) [reply pkg/nfd-worker/nfd-worker.go:51]: this import name convention is being recommended by both VIM-GO and VSCODE go plugins, should we keep it?

Comment (marquiz) [reply pkg/nfd-worker/nfd-worker.go:51]: #458 removes that all together (if we agree to merge that) so I would not change this in order to avoid unnecessary rebases.

Comment (marquiz) [reply pkg/nfd-worker/nfd-worker.go:51]: Ah, problem solved. #458 was merged 😄

Comment (marquiz) [reply pkg/nfd-worker/nfd-worker.go:51]: Although now you need to rebase 😬

Comment (ArangoGutierrez) [reply pkg/nfd-worker/nfd-worker.go:51]: hahahaha ok rebasing now

Comment (ArangoGutierrez) [reply pkg/nfd-worker/nfd-worker.go]: done, now working on the commit message

**Comment by ArangoGutierrez** [comment]
Ready for re-review @marquiz

**Review by marquiz — APPROVED** [review]
Nice improvement in the coherency of log message, @ArangoGutierrez!
/approve
